Epic Squad 1 (#8)
tryber/hackatrybe-covid: src/remote_work/markdown/interacoes.md

> # Dinâmicas de Interação

Reviewer: ```suggestion
# Dinâmicas de Interação

```

> No contexto _remoto_, onde é limitada a capacidade de sentir as estudantes, seja pela distância estrutural ou por uma carência de respostas cognitivas; as dinâmicas de interação se tornam muito importantes. Para manter atenção, engajamento e entender como estão absorvendo o conteúdo apresentado.

Reviewer: ```suggestion
No contexto _remoto_, onde é limitada a capacidade de sentir as estudantes, seja pela distância estrutural ou por uma carência de respostas cognitivas; as dinâmicas de interação se tornam muito importantes para manter atenção, engajamento e entender como estão absorvendo o conteúdo apresentado.
```

> Moore criou a expressão, distância transacional para nomear esse espaço de comunicação entre o professor e o aluno, atribuindo a ele uma dimensão psicológica. A distância transacional é um espaço a ser transposto, nas palavras de Moore. (citação)

Reviewer: ```suggestion
Moore criou a expressão distância transacional para nomear esse espaço de comunicação entre o professor e o aluno, atribuindo a ele uma dimensão psicológica. A distância transacional é um espaço a ser transposto, nas palavras de Moore. (citação)
```

Where's the reference? 🤔

Author: The review is in Betina's link that Lucas said to us to put on the page, but I think we forgot the link.

> A distância transacional é obstáculo enfrentado pela educação remota, é importante citar três pontos  nesse conceito. Veja que o objetivo dessa explicação é somente de explicar conceitos que abrangem a educação remota e mostrar ferramentas que agem a respeito desses assuntos.

Reviewer: ```suggestion
A distância transacional é um obstáculo enfrentado pela educação remota. É importante citar três pontos  nesse conceito. Veja, também, que o objetivo dessa explicação é explicar conceitos que abrangem a educação remota e mostrar ferramentas que agem acerca desses assuntos.
```

O diálogo: quanto mais possibilidade de interação, menor consciência da distância transacional. Quando há esse espaço, a atenção do estudante é mais voltada para aula.

> Perguntas sobre o conteúdo são sempre bem-vindas, mas merecem uma organização por relevância. Uma plataforma que ajuda nisso é o Slido, que ordena as perguntas feitas pelo alunos pelas mais votadas e pela ordem de postagem.

Reviewer: ```suggestion
Perguntas sobre o conteúdo são sempre bem-vindas, mas merecem uma organização por relevância. Uma plataforma que ajuda nisso é o Sli.do, que ordena as perguntas feitas pelo alunos pelas mais votadas e pela ordem de postagem.
```

> Slido : https://www.sli.do/

Reviewer: ```suggestion
Sli.do : https://www.sli.do/
```

> O professor cria a sala no site e compartilha o link para os alunos, afim de receber suas perguntas. Ao criar uma pergunta ela ser anônima, é importante tanto que as perguntas sejam feitas quanto nomeadas.

Reviewer: ```suggestion
O professor cria a sala no site e compartilha o link para os alunos, afim de receber suas perguntas. Ao criar uma pergunta ela pode ser anônima, então é importante que tanto que as perguntas sejam feitas quanto que sejam nomeadas.
```

> Compartilhar a tela é uma forma de trazer um conteúdo, que está no seu computador, reproduzido na tela de todos os alunos. Conteúdos, como slides e exemplos podem ser tanto compartilhados com estudades via à tela compartilhada, como com um _link_ de acesso.

Reviewer: ```suggestion
Compartilhar a tela é uma forma de trazer um conteúdo, que está no seu computador, reproduzido na tela de todos os alunos. Conteúdos, como slides e exemplos, podem ser tanto compartilhados com estudantes via tela compartilhada, como com um _link_ de acesso.
```

> A estrutura relacionada a aula, aproxima os alunos quanto mais flexivel e distancia quanto mais rígida. Ou seja, quando dado o incentivo para o estudante de se engajar com a classe, menores as chances que ele recorra a uma distração, claramente ao alcance.

Reviewer: ```suggestion
A estrutura da aula aproxima os estudantes quanto mais flexível for, e os distancia quanto mais rígida. Ou seja, quando dado o incentivo para o estudante se engajar com a classe, menores as chances que ele recorra a uma distração, claramente ao alcance.
```

> A autonomia do aluno entra em pauta, ela se relaciona a liberdade tomar decisões, se exige maior quando a estrutura rígida e menor diálogo. A capacidade do aluno para assumir a responsabilidade influência muito autonomia incentivar isso é uma grande desafio.

Reviewer: ```suggestion
A autonomia do estudante entra em pauta: ela se relaciona a liberdade tomar decisões, se exige maior quando a estrutura for mais rígida e o diálogo for menor. A capacidade do aluno para assumir a responsabilidade influência muito a autonomia. Incentivar isso é um grande desafio.
```

> Um outro fato que influência o envolvimento do aluno na aula é uma presença social, mesmo que ela física na atual circuntância não seja possível. É clara a presença social quando não há barreiras de distância e engajamento em atividades de interesse comum a um grupo. A interação com os colegas ganha maior importância, atividades de colaboração com grande interação entre o grupo contribuem a percepção de presença social.

Reviewer: ```suggestion
Um outro fato que influência o envolvimento do estudante na aula é uma presença social, mesmo que ela, física, na atual circunstância não seja possível. É clara a presença social quando não há barreiras de distância e quando há engajamento em atividades de interesse comum a um grupo. A interação com os colegas ganha maior importância, e atividades de colaboração com grande interação entre o grupo contribuem a percepção de presença social.
```

> Uma ferrameta que ajuda na interação é o Slack, uma plataforma de organização interna para grupos, como um WhatsApp para times ou corporações. Existem tanto conversas particulares quanto em grupo, marcar pessoas, compartilhar arquivos, canais que podem se organizar temas diversos.

Reviewer: ```suggestion
Uma ferrameta que ajuda na interação é o Slack, uma plataforma de organização interna para grupos, como um WhatsApp para times ou corporações. Existem tanto conversas particulares quanto em grupo, sendo possível marcar pessoas, compartilhar arquivos, e há canais que podem se organizar em temas diversos.
```

> Aqui esta o guia oficial: https://get.slack.help/hc/pt-br/categories/360000049043

Reviewer: ```suggestion
Aqui está o guia oficial: https://get.slack.help/hc/pt-br/categories/360000049043
```
